### PR TITLE
Di 187/grpc includes

### PR DESCRIPTION
**Pull request by juliette-grantham**
- TxsRequest and TxsReply shared between newTxs and pendingTxs
- allow using includes as a param to the script (--include tx_hash,tx_contents.v,tx_contents.from)

### CODE REVIEW

File: go/go.mod [hunk 1]
```diff
@@ -5,6 +5,7 @@ go 1.18
 require (
 	github.com/ethereum/go-ethereum v1.10.18
 	github.com/gorilla/websocket v1.5.0
+	github.com/influxdata/influxdb v1.8.3
```
Comment (daniellemachpud) [inline go/go.mod:8]: why did you change it?

File: go/go.sum [hunk 2]
```diff
@@ -192,6 +192,7 @@ github.com/huin/goutil v0.0.0-20170803182201-1ca381bf3150/go.mod h1:PpLOETDnJ0o3
 github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6/go.mod h1:aSSvb/t6k1mPoxDqO4vJh6VOCGPwU4O0C2/Eqndh1Sc=
 github.com/inconshreveable/mousetrap v1.0.0/go.mod h1:PxqpIevigyE2G7u3NXJIT2ANytuPF1OarO4DADm73n8=
 github.com/influxdata/flux v0.65.1/go.mod h1:J754/zds0vvpfwuq7Gc2wRdVwEodfpCFM7mYlOw2LqY=
+github.com/influxdata/influxdb v1.8.3 h1:WEypI1BQFTT4teLM+1qkEcvUi0dAvopAI/ir0vAiBg8=
```
Comment (daniellemachpud) [inline go/go.sum:195]: why did you change it?

**Review by daniellemachpud — APPROVED** [review]
did you try to run a compare script? checked if the performance has effected by this parameter?

**Comment by juliette-grantham** [comment]
> did you try to run a compare script? checked if the performance has effected by this parameter?

Yes, I posted about it in the dev infra channel last week: "Speed is improved for blocks feed without contents, but still 31us slower that ws according to my tests." And "Saw little to no improvement with the tx includes."
